tony-history-server/startTHS.sh

```diff
@@ -181,10 +177,25 @@ save () {
     for i do printf %s\\n "$i" | sed "s/'/'\\\\''/g;1s/^/'/;\$s/\$/' \\\\/" ; done
     echo " "
 }
-JAVA_OPTS=$(save "$@")
+
+# Set up TONY_CONF_DIR if not exist
+if [[ -z "${TONY_CONF_DIR}" ]]; then
+  echo "TONY_CONF_DIR not set. Default to /export/apps/tony"
+  TONY_CONF_DIR="/export/apps/tony"
+else
+  TONY_CONF_DIR="${TONY_CONF_DIR}"
```

Comment:
Is this `else` block necessary? Looks like an identity. Could you just proceed with using `$TONY_CONF_DIR` below without this assignment?

Reply:
Also including this in https://github.com/linkedin/TonY/pull/103.